packages/pds/src/account-manager/db/schema/app-password.ts

```diff
@@ -3,6 +3,7 @@ export interface AppPassword {
   name: string
   passwordScrypt: string
   createdAt: string
+  privileged: 0 | 1 | null
```

Comment:
What does the `null` do for us?

Reply:
I don't think you can make new "non-nullable" columns in sqlite unfortunately so we're just stuck with the `null` as a result of the migration :/

Reply:
(may be missing some sqlite feature that allows this, but that was the case the last time i checked)

Reply:
Huh, I really thought you could!  If you can, you would def need to set a default at the same time.

Reply:
e.g.
```sql
ALTER TABLE app_password
  ADD COLUMN privileged NOT NULL DEFAULT 0;
```

Reply:
oh whatdya know, you can as long as you supply a default. good catch 👍